extensions/things/src/components/TodoListItemActions.tsx

```diff
@@ -208,8 +208,26 @@ New title:
             title="Date…"
             icon={Icon.Calendar}
             min={new Date()}
-            onChange={(date) => schedule(date ? date.toISOString() : 'anytime')}
-            type={Action.PickDate.Type.Date}
+            onChange={(date) => {
+              if (date) {
```

Comment:
Minor: prefer returning edge cases early over `if`/`else`. E.g.:

```typescript
if (!date) {
  schedule('anytime')
  return
}

// handle date
```

Reply:
Makes sense to me! Updated with this commit – https://github.com/raycast/extensions/pull/18771/commits/213a7beb6c61fabe6f8fabb8089fc5b13aa3b1dc ✌️